packages/manager/apps/pci-load-balancer/src/api/data/flavors.ts

```diff
@@ -1,17 +1,17 @@
 import { v6 } from '@ovh-ux/manager-core-api';
-import { TAddon } from '@/pages/create/store';
 import { TFlavor } from '@/api/data/load-balancer';
+import { TProductAddonDetail } from '@/types/product.type';
 
 export const getFlavor = async (
   projectId: string,
   regionName: string,
-  addon: TAddon,
+  product: TProductAddonDetail,
 ): Promise<TFlavor> => {
   const { data } = await v6.get<TFlavor[]>(
     `/cloud/project/${projectId}/region/${regionName}/loadbalancing/flavor`,
   );
 
   return data.find(
-    (regionalizedFlavors) => regionalizedFlavors.name === addon.technicalName,
+    (regionalizedFlavors) => regionalizedFlavors.name === product.technicalName,
```

Comment:
This should not be the place to compute data, but rather than in a the dedicated hook